src/com/lftechnology/java/training/alina/basic/fibonacci/EvenFibonacci.java

```diff
@@ -4,31 +4,27 @@
 import java.util.logging.Logger;
 
 /**
- * Each new term in the Fibonacci sequence is generated by adding the previous
- * two terms. By starting with 1 and 2, the first 10 terms will be: 1, 2, 3, 5,
- * 8, 13, 21, 34, 55, 89, ... By considering the terms in the Fibonacci sequence
- * whose values do not exceed four million. Find the sum of the even-valued
- * terms.
+ * Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will
+ * be: 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ... By considering the terms in the Fibonacci sequence whose values do not exceed four million.
+ * Find the sum of the even-valued terms.
  * 
  * @author Alina Shakya<alinashakya@lftechnology.com>
  */
 public class EvenFibonacci {
 
-	private static final Logger LOGGER = Logger.getLogger(EvenFibonacci.class
-			.getName());
+    private static final Logger LOGGER = Logger.getLogger(EvenFibonacci.class.getName());
 
-	private EvenFibonacci() {
-		// TODO Auto-generated constructor stub
-	}
-	
-	public static void main(String[] args) throws Exception {
-		try {
-			SumOfEvenFibonacci fibonacci = new SumOfEvenFibonacci(0, 1);
-			LOGGER.log(Level.INFO, "The sum of the even-valued fibonacci terms : {0}", new Object[] {fibonacci.getSumOfEvenFibonacci()});
-		} catch (ArithmeticException ex) {
-			LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] {ex.getMessage()});
-			throw new Exception();
-		}
-	}
+    private EvenFibonacci() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        try {
+            SumOfEvenFibonacci fibonacci = new SumOfEvenFibonacci(0, 1);
+            LOGGER.log(Level.INFO, "The sum of the even-valued fibonacci terms : {0}", new Object[] { fibonacci.getSumOfEvenFibonacci() });
+        } catch (ArithmeticException ex) {
+            LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] { ex.getMessage() });
+            throw new Exception();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)